applications/vessel-history/src/features/profile/components/activity/Activity.module.css

```diff
@@ -18,6 +18,10 @@
   padding: 0 var(--space-S);
 }
 
+.event .eventData {
+  overflow: hidden;
```

Comment:
add this will looks better https://developer.mozilla.org/es/docs/Web/CSS/text-overflow